src/lib/models/types.ts

```diff
@@ -202,23 +198,22 @@ function wrap(type: Type, bp: number) {
 export class ArrayType extends Type {
     override readonly type = "array";
 
-    /**
-     * The type of the array elements.
-     */
-    elementType: Type;
-
-    constructor(elementType: Type) {
+    constructor(
+        /**
+         * The type of the array elements.
+         */
```

Comment:
I'd rather not do this as a matter of style - either get rid of the comment, or keep the declaration and parameter separate. Mixing parameters and comments makes the code harder to scan quickly

Reply:
Makes sense. I've moved this to the constructor documentation.